docs/components/README.md

```diff
@@ -259,68 +260,73 @@ export default {
   deactivated() {},
   beforeDestroy() {},
   destroyed() {},
-  errorCaptured() {},
-}
+  errorCaptured() {}
+};
 ```
 
-### Propriétés d'instance de vue
+### Vue instance properties
+
+::: tip
+`vm` is often used as a convention to refer to a Vue component instance
+:::
 
 - `vm.$data`
 - `vm.$props`
 - `vm.$slots`
 - `vm.$refs`
 - `vm.$listeners`
-- `vm.$options`: les options passées au composant
-- `vm.$el`: référence à l'élément HTML racine sur lequel le composant est monté
-- `vm.$parent`: composant parent
-- `vm.$root`: composant racine
-- `vm.$children`: composants enfant
-
-### Méthodes d'instance de vue
-
-- `vm.$watch`: déclare programmatiquement un watcher
-- `vm.$set`: assigne une donnée en forçant la réactivité
-- `vm.$delete`: supprime une donnée en forçant la réactivité
-- `vm.$on`: déclare programmatiquement un listener
-- `vm.$once`: déclare un listener avec le modificateur once
-- `vm.$off`: supprime un listener
-- `vm.$emit`: émet un événement
-- `vm.$mount`: attache le composant à un élément du DOM
-- `vm.$destroy`: supprime l'instance de composant
-- `vm.$forceUpdate`: force la mise à jour complète du composant (à éviter)
-- `vm.$nextTick`: reporte l'exécution d'une fonction au prochain tick (boucle d'événements)
-
-## TP: Décomposer son application
-
-1. Refactorez le code existant en créant un composant `Film.vue` servant à afficher les détails d'un film. Ajoutez des `props` pour passer les données de chaque film au composant.
-2. Créer un autre composant `SearchFilm.vue` contenant un formulaire de recherche ainsi que la liste de `Film` en résultats dessous:
+- `vm.$options`: all the component options
+- `vm.$el`: reference to the root HTML element on which the component is mounted
```

Comment:
il faut accorder à chaque fois non ? `references`,  `declares`, `assigns`, etc.

Reply:
reference je l'utilisais comme nom: a reference to

pour les autres verbes je pensais à l'infinitif quand je l'ai écrit, mais je peux mettre l'indicatif partout

Reply:
je vais mettre infinitif partout comme sur la doc officielle

Reply:
d'acc parfait ! :+1: